website/addons/osfstorage/model.py

```diff
@@ -218,6 +241,20 @@ def path(self):
     def node(self):
         return self.node_settings.owner
 
+    @property
+    def rented(self):
```

Comment:
The rented function is meant to return the user id as a string so that the js part can compare it to the current user. It could be right to return None instead of an empty string, by the renter function could not return a bool unless the user was an input